api/dsap/search.js

```diff
@@ -2,16 +2,26 @@
 module.exports = function (context) {
     const models =  context.models;
 
-    function searchArtists(searchString, options, relation) {
-        return models.artists.findAll({
+    function searchArtists(searchString, options) {
+        if (searchString != '') {
+            searchString = '%' + searchString + '%';
+        } else {
```

Comment:
This is a little bit an odd one out. Why does this have other semantics and a different query then let's say `findAllReleases`?